packages/server/routes/api/users.js

```diff
@@ -58,24 +58,36 @@ router.get(
 router.get(
   "/:userId/guilds",
   AuthController.authenticate,
+  [
+    param("userId")
+      .notEmpty()
+      .withMessage("User ID cannot be empty")
+      .isUUID()
+      .withMessage("Invalid user ID provided")
+      .custom(async (value) => {
+        const user = await UserController.getUser(value);
+        if (!user) {
+          throw new Error("User with ID ${value} not found");
+        }
+        return true;
+      }),
+  ],
```

Comment:
We don't want to clutter the route handler files with all the validators for our routes, so move this to `packages/server/validators/users.js` and export it to use here instead. You can look at the guild validators in main branch for reference on how to structure the file.